### PR TITLE
[2.6_WAS] Update to ASM 9.1

### PR DESCRIPTION
**Pull request by jgrassel**
Updating ASM level on 2.6_WAS to 9.1 -- opted not to try and change to the 2.7 branch's plugin method at this time.

### CODE REVIEW

File: moxy/org.eclipse.persistence.moxy/.classpath [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<classpathentry kind="lib" path="resource"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/javax.ws.rs_1.1.1.v20101004-1200.jar" sourcepath="/ECLIPSELINK_HOME/plugins/javax.ws.rs.source_1.1.1.v20101004-1200.jar"/>
-	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm_9.0.0.v202010221051.jar" sourcepath="/ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm.source_9.0.0.v202010221051.jar"/>
+	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/org.eclipse.persistence.9.1.0.v202105140933.jar" sourcepath="/ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm.source_9.1.0.v202105140933.jar"/>
```
Comment (dazey3) [inline moxy/org.eclipse.persistence.moxy/.classpath:7]: I think maybe this name is wrong

**Review by dazey3 — CHANGES_REQUESTED** [review]
Just one minor issue. Everything else LGTM

File: moxy/org.eclipse.persistence.moxy/.classpath (hunk 1 above)
Comment (jgrassel) [reply moxy/org.eclipse.persistence.moxy/.classpath:7]: Yeah that's wrong, thanks for catching that.

Comment (jgrassel) [reply moxy/org.eclipse.persistence.moxy/.classpath]: Should be fixed now.

File: moxy/org.eclipse.persistence.moxy/.classpath [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	<classpathentry kind="lib" path="resource"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/javax.ws.rs_1.1.1.v20101004-1200.jar" sourcepath="/ECLIPSELINK_HOME/plugins/javax.ws.rs.source_1.1.1.v20101004-1200.jar"/>
-	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm_9.0.0.v202010221051.jar" sourcepath="/ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm.source_9.0.0.v202010221051.jar"/>
+	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm.9.1.0.v202105140933.jar" sourcepath="/ECLIPSELINK_HOME/plugins/org.eclipse.persistence.asm.source_9.1.0.v202105140933.jar"/>
```
Comment (dazey3) [inline moxy/org.eclipse.persistence.moxy/.classpath:7]: Should it be `asm_9.1.0.` instead of `asm.9.1.0.`?

**Review by dazey3 — CHANGES_REQUESTED** [review]

File: moxy/org.eclipse.persistence.moxy/.classpath (hunk 2 above)
Comment (jgrassel) [reply moxy/org.eclipse.persistence.moxy/.classpath:7]: oh right.

Comment (jgrassel) [reply moxy/org.eclipse.persistence.moxy/.classpath]: fixed

**Review by dazey3 — APPROVED** [review]
LGTM! Thanks Joe

**Pull request merged**